Tests/MapboxMapsTests/Snapshot/SnapshotterTests.swift

```diff
@@ -28,19 +28,27 @@ final class SnapshotterTests: XCTestCase {
     }
 
     // Test snapshot start invokes mockMapSnapshotter startStub
-    func testSnapshotterStart() {
+    func testSnapshotterStartInvocation() {
+        // asserts that the mock and snapshotter reach start
+        snapshotter.start(overlayHandler: nil) { (result) in
```

Comment:
With MockMapSnapshotter just swallowing calls(including this one) - does completion gets invoked here?